app/scripts/controllers/landingPage.js

```diff
@@ -51,10 +52,20 @@ angular.module('openshiftConsole')
 
     AuthService.withUser().then(function() {
       var includeTemplates = !_.get(Constants, 'ENABLE_TECH_PREVIEW_FEATURE.template_service_broker');
-      Catalog.getCatalogItems(includeTemplates).then(function(items) {
+      Catalog.getCatalogItems(includeTemplates).then(_.spread(function(items, errorMessage) {
+        if (errorMessage) {
+          var alert = {
+            name: 'error-loading-catalog-items',
+            data: {
+              type: 'error',
+              message: errorMessage
+            }
+          };
+          AlertMessageService.addAlert(alert);
```

Comment:
Adding the alert to `AlertMessageService` will show the message on the next page the user navigates to, which I don't think is what we want.

I suggest we change the `Catalog` service to pass `{ errorNotification: false }` and then show its formatted message here. In the meantime, we could either not do anything with the message or possibly just show separate alerts until we make the catalog change?